draft-ietf-gnap-core-protocol.md

```diff
@@ -3830,6 +3821,17 @@ NOTE: '\' line wrapping per RFC 8792
 }
 ~~~
 
+Key proofing methods SHOULD define a mechanism to allow the rotation of keys discussed
+in {{rotate-access-token-key}}. Key rotation mechanisms MUST define a way for presenting
+proof of two keys simultaneously with the following attributes:
+
+- The value of or reference to the new key material MUST be signed by the existing key.
```

Comment:
This is a bit redundant with the part in section 6.1.1 (although a bit clearer here)